src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -187,7 +187,9 @@ def _are_provided_values_valid(self, values):
         valid_fields = self._get_valid_fields()
         if self.attr in valid_fields:
             for value in values:
-                if value not in valid_fields[self.attr]:
+                if any([x.casefold() == value.casefold() for x in valid_fields[self.attr]]):
+                    ret = True
```

Comment:
`ret` does not exist anymore

Reply:
This was done and changed to all() using a !=